teemaderegister-be/src/controllers/users.js

```diff
@@ -174,8 +171,19 @@ module.exports.resetPicture = async (req, res) => {
     { new: true }
   ).select('profile.image.full updatedAt')
 
-  await fs.unlink(`${process.env.UPLOAD_DIR + original}`, err =>
-    log.debug(`${err && err.code} - Probably ${original} already deleted`))
+  await fs.unlink(`${process.env.UPLOAD_DIR + original}`, (err) =>
+    log.debug(`${err && err.code} - Probably ${original} already deleted`)
+  )
 
   return res.json({ user, message: 'Picture updated successfully' })
 }
+
+module.exports.getAllUsers = async (req, res) => {
+  const users = await User.find({})
+
+  let data = []
+  for (const number in users) {
+    data.push(users[number]['profile'])
+  }
+  return res.json(data)
```

Comment:
I would return data format as a user object as it is returned from DB, still including a unique identifier. Modifying user object too much is not a too good idea.